boards/x86/quark_se_c1000_devboard/quark_se_c1000_devboard.dts

```diff
@@ -54,4 +54,12 @@
 
 &spi1 {
 	status = "ok";
+
+	cc2520@0 {
+		 compatible = "ti,cc2520";
+		 reg = <0x0>;
+		 spi-max-frequency = <8000000>;
+		 cs-gpios = <&gpio0 11 0>;
+		 label = "CC2520";
+	};
```

Comment:
@lpereira I guess that's what you would be interested to, as we talked in device driver API meeting